### ISSUE TITLE
Some recommended style issues

### ISSUE THREAD

**Issue by ocaisa** [E-CAM/jobqueue_features#6]
@dwhswenson raised some style issues when reviewing the code:

there are a lot of kwarg updates that might be cleaned up with a function like

```python
# instead of making a dictionary to update with all at once, call update_kwargs for each name
def update_kwargs(kwargs, obj, name, default):
    kwargs.update({name: getattr(obj, name, getattr(kwargs, name, default))})
```

The idiom with the nested `getattr`s is repeated several times, and requires the use of `name` 3 times. It is also the source of some of the longer lines (not to mention some harder-to-read and somewhat inconsistent formatting from black).

**Comment by dwhswenson** [comment]
To clarify, the sorts of things that I thought were inconsistent in black's formatting are lines like these:

https://github.com/E-CAM/jobqueue_features/blob/8d3725f36af2c93b70ff349437f1909776afeab9/jobqueue_features/decorators.py#L126-L133

In the first dict entry, all the arguments to `getattr` are on one line. In the second entry, each argument has its own line. It takes a second to see that these are all doing essentially the same thing; putting the behavior in a function would make it more readable, in my opinion. Also, easier to write: only need to include the string for the name once, instead of 3 times.

**Comment by AdamWlodarczyk** [comment]
Agree. String literals should be introduced once.

**Comment by ocaisa** [comment]
This is also resolved in #9
